src/java.base/share/classes/jdk/internal/access/JavaLangAccess.java

```diff
@@ -273,9 +273,10 @@ public interface JavaLangAccess {
     void addEnableNativeAccessAllUnnamed();
 
     /**
-     * Returns true if module m can access restricted methods.
+     * Ensure that the given module has native access. If not, warn
```

Comment:
We have a section on restricted methods in the foreign package javadoc. Perhaps we could link to that?